frontend/src/components/Navbar/NavbarWorksheetSearch.tsx

```diff
@@ -252,10 +274,92 @@ export function NavbarWorksheetSearch() {
                 placeholder="Main Worksheet"
                 onChange={(selectedOption: ValueType<Option, boolean>) => {
                   if (isOption(selectedOption)) {
+                    if (selectedOption.value in ['0', '1', '2', '3']) {
+                      setDeleteBool(true);
+                    } else {
+                      setDeleteBool(false);
+                    }
```

Comment:
This entire thing is `setDeleteBool(['0', '1', '2', '3'].includes(selectedOption.value)`